pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -669,19 +669,7 @@ func (h *Helper) ServerPrereleaseVersion() (semver.Version, error) {
 		return semver.Version{}, fmt.Errorf("did not find version in command output")
 	}
 
-	// The OSE version may have > 4 parts to the version string
-	// We'll only take the first 3
-	parts := strings.Split(versionStr, ".")
-	if len(parts) > 3 {
-		versionStr = strings.Join(parts[:3], ".")
-	}
```

Comment:
@csrwng i'm hoping this can be removed because otherwise "3.6.0-alpha.2" and "3.6.0-alpha.3" are indistinguisable (the .2 and .3 are getting dropped)

What was the motivation for doing this originally?